fv3_cap.F90

```diff
@@ -1003,6 +1011,11 @@ subroutine InitializeRealize(gcomp, rc)
 
     if (ESMF_LogFoundError(rcToCheck=urc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__, rcToReturn=rc)) return
 
+    timers = 0.
+    timere = 0.
```

Comment:
timep2re should also be initialized, because on the very first call of  ModelAdvance_phase1 it will be uninitialized. 

Also if these values are initially zero, on the very first call of ModelAdvance* the elapsed time will not be correct. Since MPI_WTime returns time since arbitrary point in the past.  Maybe if these values are zero you can skip printing initial time difference between the current and the previous time.

Reply:
Thanks.

Well, strictly speaking all 'start' times (timers, timep1rs, timep2rs) can actually be declared as local variables, they do not have to be saved at the module level, since they are only used locally in three respective 'advance' routines. 

timep1re can also be local in ModelAdvance_phase1. Only timere and timep2re must be saved at the module level.

I think the code is now correct, so these are just suggestions.

Reply:
I was thinking of that too. The code is updated.